src/Maestro/Maestro.MergePolicies/ForwardFlowMergePolicy.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Maestro.MergePolicyEvaluation;
+using Microsoft.DotNet.DarcLib;
+using Microsoft.DotNet.DarcLib.Models.VirtualMonoRepo;
+
+namespace Maestro.MergePolicies;
+internal class ForwardFlowMergePolicy : MergePolicy
+{
+    public override string DisplayName => "ForwardFlow";
+
+    private static readonly string configurationErrorsHeader = """
+         ### :x: Check Failed
+
+         The following error(s) were encountered:
+
+
+        """;
+
+    private static readonly string seekHelpMessage = """
+
+
+        ### :exclamation: IMPORTANT
+
+        The `source-manifest.json` and `Version.Details.xml` files are managed by Maestro/darc. Outside of exceptional circumstances, these files should not be modified manually.
```

Comment:
Here it's also missing the paths but also all the other texts should reference `VersionFiles` and `VmrInfo` classes where the paths to these files are defined. We should not hardcode those.